src/main/java/uk/gov/hmcts/reform/et/syaapi/config/OpenAPIConfiguration.java

```diff
@@ -14,7 +14,7 @@
     public OpenAPI openAPI() {
         return new OpenAPI()
             .info(new Info().title("et-sya-api")
-                     .license(new License().name("MIT").url("https://opensource.org/licenses/MIT")))
+                      .license(new License().name("MIT").url("https://opensource.org/licenses/MIT")))
```

Comment:
Can we move the hard coded string to constant file to start using it? If we use this in multiple places it would be one point change in future.

Reply:
Created new constants file